include/xtensor/xaccumulator.hpp

```diff
@@ -25,18 +25,33 @@ namespace xt
 
 #define DEFAULT_STRATEGY_ACCUMULATORS evaluation_strategy::immediate_type
 
+    namespace detail
+    {
+        template <class V = void>
+        struct accumulator_identity: public xtl::identity
```

Comment:
structure inheritance is `public` by default, no need to specify it

Reply:
done